variables.tf

```diff
@@ -30,3 +32,70 @@ variable "scan_on_push" {
   description = "Scan image on push to repo."
   default     = true
 }
+
+# ECS variables
+
+variable "app_name" {
+  type        = string
+  description = "Name of the application"
+}
+
+variable "environment" {
+  type        = string
+  description = "Environment name"
+}
+
+variable "task_name" {
+  type        = string
+  description = "Name of the task to be run"
+}
```

Comment:
Do we really need to pass all this in, or can we do it with just `environment`?  Or can we use other variables we're already passing, like github repo name?

Reply:
removed app_name and task_name, filling in with just github_repo_name